dashboard/api/pkg/store/queries/get_job_input_relations.sql

```diff
@@ -0,0 +1,42 @@
+WITH job_inputs AS (
+    SELECT
+        id AS JobID,
+        input_element ->> 'StorageSource' as storage_source,
+        input_element ->> 'CID' AS input
+    FROM
+        job,
+        json_array_elements(jobdata::json -> 'Spec' -> 'inputs') AS input_element
+    WHERE input_element ->> 'StorageSource' = 'IPFS'
```

Comment:
should we also include estuary and filecoint/lotus sources?

Reply:
I wasn't able to find any jobs using the Filecoin/Lotus or Esturary `StorageSource` in the database.
i.e. the below query returns zero results
```sql
SELECT
    id AS JobID,
    input_element ->> 'StorageSource' as storage_source,
    input_element ->> 'CID' AS input
FROM
    job,
    json_array_elements(jobdata::json -> 'Spec' -> 'inputs') AS input_element
WHERE job.apiversion = 'V1beta1' and
      input_element ->> 'StorageSource' != 'IPFS' and
      input_element ->> 'StorageSource' != 'URLDownload' and
      input_element ->> 'StorageSource' != 'LocalDirectory' and
      input_element ->> 'StorageSource' != 'Inline'
```
I didn't feel confident including these sources as I am unsure what the json schema looks like. Alright if I leave this as IPFS only?

---------------
At some point we should probably also be plotting this data: aggregate counts of different storage sources.